Core/Utilities/ConsumerTools.py

```diff
@@ -0,0 +1,21 @@
+"""A set of tools for the consumer system.
+   In the future the tools corresponding to the CS operations
+   probably should be extracted along with other helpers
+   to create a common block of CS reading operationg
+   e.g. with PathFinder.py, Registry.py and CSHelper.py
+   but also in AgentModule.py and RequestHandler.py
+"""
+
+from DIRAC import gConfig
+from DIRAC.ConfigurationSystem.Client import PathFinder
+
+def getOption( optionName, defaultValue = None ):
```

Comment:
are these two methods really useful ? This just forward the exact same arguments, I don't see why the caller could not call directly gLoader ?